src/Modules/Trading/Containers/trade.jsx

```diff
@@ -1,18 +1,30 @@
-import PropTypes             from 'prop-types';
-import React                 from 'react';
-import ChartLoader           from 'App/Components/Elements/chart-loader.jsx';
-import UILoader              from 'App/Components/Elements/ui-loader.jsx';
-import { connect }           from 'Stores/connect';
-import PositionsDrawer       from 'App/Components/Elements/PositionsDrawer';
+import PropTypes            from 'prop-types';
+import React                from 'react';
+import ChartLoader          from 'App/Components/Elements/chart-loader.jsx';
+import UILoader             from 'App/Components/Elements/ui-loader.jsx';
+import { connect }          from 'Stores/connect';
 import MarketIsClosedOverlay from 'App/Components/Elements/market-is-closed-overlay.jsx';
-import NotificationMessages  from 'App/Containers/notification-messages.jsx';
-import Digits                from 'Modules/Contract/Components/Digits';
-import InfoBox               from 'Modules/Contract/Components/InfoBox';
-import Test                  from './test.jsx';
-import FormLayout            from '../Components/Form/form-layout.jsx';
-import { isDigitTradeType }  from '../Helpers/digits';
+import Digits               from 'Modules/Contract/Components/Digits';
+import InfoBox              from 'Modules/Contract/Components/InfoBox';
+import Test                 from './test.jsx';
+import FormLayout           from '../Components/Form/form-layout.jsx';
+import { isDigitTradeType } from '../Helpers/digits';
 
-const SmartChart = React.lazy(() => import(/* webpackChunkName: "smart_chart" */'../../SmartChart'));
+const SmartChart           = React.lazy(() => import(/* webpackChunkName: "smart_chart" */'../../SmartChart'));
+const NotificationMessages = React.lazy(() => import(/* webpackChunkName: "notification-messages" */'App/Containers/notification-messages.jsx'));
+
+const LazyPositionsDrawer = ({ should_load }) => {
+    if (should_load) {
+        const PositionsDrawer = React.lazy(() => import(/* webpackChunkName: "positions-drawer" */'App/Components/Elements/PositionsDrawer'));
```

Comment:
Maybe can rename this to `PositionsDrawerChunk` and rename `LazyPositionsDrawer` back to `PositionsDrawer`?

Reply:
I see, sure. Why not.